aries-site/src/layouts/content/Example/CompareUseCases.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, ResponsiveContext, Text } from 'grommet';
+import { Close, Checkmark } from 'grommet-icons';
+
+export const CompareUseCases = ({ imageSrcGood, imageAltGood, captionGood, imageSrcBad, imageAltBad, captionBad }) => {
+
+    const size = useContext(ResponsiveContext);
+    let direction;
+    if (size !== 'small') {
+        direction = 'row';
+    } else direction = 'column';
+
+    let width;
+    if (size !== 'small') {
+        width = 'medium';
+    } else width = '100%';
+
+    return (
+        <Box
+            direction={direction}
+            gap="large"
+            pad={{ vertical: 'medium' }}>
+            <Box width={width}>
+                <Image src={imageSrcGood} alt={imageAltGood} />
+                <Box
+                    direction="row"
+                    pad={{ top: 'small' }}>
```

Comment:
Prefer to use `gap` in containing Box to control spacing and `gap` in this Box to control spacing (if needed) between its children.

```suggestion

```